### PR TITLE
prepared release of version 2.8.1

### PR DESCRIPTION
**Pull request by hsm207**
**Proposed changes**:
-  Update urllib3 to 1.26.5 to fix CVE-2021-33503

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.mdx [hunk 1]
```diff
@@ -17,6 +17,9 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.1] - 2021-07-20 No significant changes.
```
Comment (tczekajlo) [inline CHANGELOG.mdx:20]: Can we put here that we updated urlib3 to the version that you mentioned in the issue description?

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: CHANGELOG.mdx (hunk 1 above)
Comment (hsm207) [reply CHANGELOG.mdx:20]: done. how does it look?

Comment (tczekajlo) [reply CHANGELOG.mdx:20]: Can we remove "No significant changes."?

**Review by tczekajlo — APPROVED** [review]
I'll approve it, I've left only one small coment

File: CHANGELOG.mdx (hunk 1 above)
Comment (hsm207) [reply CHANGELOG.mdx]: @tczekajlo done!

**Pull request merged**